Simulation/Save_Var_to_File.m

```diff
@@ -0,0 +1,16 @@
+function Save_Var_to_File(var,file_name)
+  
+  filename = [file_name ".txt"] ;%adds .txt to the file's name
```

Comment:
Is the .txt extension necessary? Are there other formats that Octave uses for their data files?

Reply:
Any file type will work. I just have to make both functions have to load/save the same file type.

Reply:
Cool just checking!